astropy/io/fits/tests/test_image.py

```diff
@@ -1132,17 +1132,26 @@ def test_comp_image_quantize_level(self):
         Test that quantize_level is used.
 
         """
-        import scipy.misc
+        import scipy
+        from astropy.utils import minversion
+
+        SCIPY_LT_1_10 = not minversion(scipy, '1.10dev0')
         np.random.seed(42)
-        data = scipy.misc.ascent() + np.random.randn(512, 512)*10
+
+        if SCIPY_LT_1_10:
+            import scipy.misc
+            scipy_data = scipy.misc.ascent()
+        else:
+            import scipy.datasets
```

Comment:
Does this download data?

Reply:
Yes, I think it downloads the "ascent" image data from their repo.

Reply:
I think? https://github.com/scipy/scipy/blob/06da9ee65cdee4a8aa036767489bfc5a030527f4/scipy/datasets/_fetchers.py#L76

Hmm, should we mark this as remote data?

Reply:
> Hmm, should we mark this as remote data?

This was my thinking, but then I am surprised it worked without that.

Reply:
https://raw.githubusercontent.com/scipy/dataset-ascent/main/ascent.dat

Reply:
@Cadair , looks better?